README.md

```diff
@@ -1 +1,131 @@
+
+**[Book Store]**
+
+> The React Book Store project is a web app built using React that lets users browse/search books, add to cart, checkout & manage their library. It has a clean UI, is scalable & responsive. Built with modern web technologies including React, Redux & Router. Ideal for learning & building similar apps.
+
+## 🛠 Built With <a name="built-with"></a>
+- [x] HTML, CSS and JavaScript
+- [x] Linters: Lighthouse, Webhint, Stylelint and ESLint
+- [x] VS Code
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- [x] A web browser like Google Chrome
+- [x] A code editor like Visual Studio Code with Git and Node.js.
+
+### Setup
+
+Clone this repository to your desired folder using the link given below:
+
+use the git clone command to clone this repository or download ZIP
+
+### Install
+
+Install this project with:
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone https://github.com/Odongo006/Bookstore.git
+```
+### Usage
+
+To run the project, execute the following command: 
+- Repository[@repo](https://github.com/Odongo006/Bookstore.git)
+
+
+
+
+
+
+### Run tests
+
+To check for linters, use the files provided by [Microverse](https://github.com/microverseinc/linters-config). A GitHub action is also set to run during pull request.
+```
+$ npm install
+```
+
+### Key Features
+- Book browsing and searching: Users can easily browse and search through a collection of books.
+
+
+### Future Features
+
+- Responsive design: The app is fully responsive, ensuring a great user experience on all devices.
+- Modern web technologies: The app is built using modern web technologies such as React, Redux, and React Router, ensuring scalability, robustness, and maintainability.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Henry**
+
+- GitHub: [@Odongo006](https://github.com/Odongo006)
+- Twitter: [@HenryOdongo007](https://twitter.com/HenryOdongo007)
+- LinkedIn: [@HenryOdongo](https://www.linkedin.com/in/henry-odongo-91b830182/)
+- Website: [@Portfolio](https://odongo006.github.io/My-Portfolio-Project/)
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing "></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Odongo006/Bookstore/issues).
+
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give it a ⭐️
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- [Microverse](https://www.microverse.org/)
+- Code Reviewers
+- Coding Partners
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [REQUIRED] This license is referencing the `MIT.md` file which is supposed to contain the license but you do not have the file so this throws a 404 error. Kindly create a new file called `MIT.md` and add a license to it. 
- NOTE: I recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.

Reply:
Hi @Whoistolu ,

Thank you for your feedback and for bringing the missing MIT.md file to my attention. I apologize for the inconvenience caused by the 404 error.

I have made the necessary changes and have created a new file called MIT.md in the root directory of the project. I have also added the MIT license to the file, as per your recommendation.

I appreciate your suggestion to use the templates available on GitHub to set up the license quickly, and will keep that in mind for future projects.

Please let me know if there is anything else that needs to be done, and thank you for your time and attention to this matter.

Reply:
Hello @Odongo006 , No need to apologize brother.
Great job you have implemented the changes and I can confirm it works now 👍🏽